src/pages/markets/components/sections/_hero.tsx

```diff
@@ -88,9 +89,15 @@ export const Hero = () => {
                     align="center"
                     mt="1.6rem"
                 >
-                    {localize(
-                        'Learn about the markets that you can trade online with Deriv, including forex, synthetic indices, stocks, stock indices, cryptocurrencies, basket indices, and commodities.',
-                    )}
+                    <EU>
+                        <Localize translate_text="Learn about the markets that you can trade online with Deriv, including forex, commodities, synthetic indices, stocks, stock indices, and cryptocurrencies." />
```

Comment:
```suggestion
                        <Localize translate_text="Learn about the markets that you can trade online with Deriv, including forex, commodities, synthetic indices, stocks & indices, and cryptocurrencies." />
```

Reply:
`stocks, stock indices` to be changed to `stocks & indices` for all 3. @vijayasree-deriv

Reply:
will confirm with @shen.

Reply:
updated as per the redmine card @sean-binary